src/SonataAdminBundle.php

```diff
@@ -61,6 +62,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new TwigStringExtensionCompilerPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 1);
         $container->addCompilerPass(new AdminMakerCompilerPass());
         $container->addCompilerPass(new AddAuditReadersCompilerPass());
+        $container->addCompilerPass(new AliasDeprecatedPublicServicesCompilerPass());
```

Comment:
```suggestion
        $container->addCompilerPass(new AliasDeprecatedPublicServicesCompilerPass(), PassConfig::TYPE_AFTER_REMOVING);
```

I think with this we can probably remove all the previous messages, the one from the screenshot.